fastscanner/services/scanners/tests/test_scan_endpoint.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from datetime import date
+from unittest.mock import AsyncMock, MagicMock, patch
+from fastapi.testclient import TestClient
+from fastapi import status
+
+from fastscanner.adapters.rest.main import app
+from fastscanner.adapters.rest.scanner import get_scanner_service
+from fastscanner.services.scanners.lib import ScannersLibrary
+
+
+@pytest.fixture
+def client_with_scanner_service(scanner_service):
+    service, channel = scanner_service
+    app.dependency_overrides[get_scanner_service] = lambda: service
+    client = TestClient(app)
+    yield client, service, channel
+    app.dependency_overrides.clear()
+
+
+def test_scan_endpoint_success(client_with_scanner_service):
```

Comment:
Instead of defining client_with_scanner_service, you can just call both fixtures here. test_...(client, scaner_service)